### ISSUE TITLE
Cannot find name 'isProd'

### ISSUE THREAD

**Issue by matheusgrieger** [jodit/jodit-react#77]
Following instructions from README, I cannot get the editor to show up at all because the file `jodit/src/core/helpers/string/i18n.ts` at `(148,7)` is throwing that error. Am I doing something wrong or is this a bug?
My implementation is as simple as this:

```tsx
```

**Comment by PhillipChaffee** [comment]
I am having the same problem. Looking into it.

**Comment by PhillipChaffee** [comment]
@matheusgrieger I am guessing you  are using Typescript as well?

**Comment by matheusgrieger** [comment]
@PhillipChaffee yes I am! No idea if this happens with JS as well or is exclusive to TS.

**Comment by joakimwennergren** [comment]
I just did an yarn upgrade and i'm seeing this issue aswell. Using typescript.

**Comment by marcus3Devel** [comment]
I'm having the same issue please fix this because it's a fantastic editor!

**Comment by alainmoran** [comment]
Me too ... for those people who did have a working version, what version were you on previously that was known to work?

I dare say this will be fixed in the future, it looks to me like some kind of linking issue since Intellij can find the definitions, but no matter what I do with my tsconfig the compiler does not.

For now though I'm happy to roll back to an older version just so that I can get going.

**Comment by alainmoran** [comment]
I just stepped back a few versions, and I found this one compiles ... I wonder if it's the 3.3->3.4 boundary that causes the problems?

```
    "jodit": "3.3.23",
    "jodit-react": "1.0.42",
```

**Comment by DaveVanRijn** [comment]
> I just stepped back a few versions, and I found this one compiles and appears to run (I've only just switched to jodit, so any errors I'm having are probably my fault now) ... I wonder if it's the 3.3->3.4 boundary that causes the problems?
> 
> ```
>     "jodit": "3.3.23",
>     "jodit-react": "1.0.42",
> ```

This version doesn't work either, because of issue #74. Version 1.0.49 seemed to fix issue #74, but also introduced this current issue.

**Comment by DaveVanRijn** [comment]
For anyone willing to use a horrible workaround:

After struggling with this for a few hours, I found that the problem is caused by using the Webpack DefinePlugin in the Jodit source, which doesn't go really well with TypeScript. 
Placing a `declare const isProd: boolean;` or `declare const appVersion: string;` in every file they are referenced by seems to fix the issue temporarily (after `@ts-ignore`-ing multiple other typing errors). Changes to the code are used by the project after rebuilding the Jodit package. My project finally builds and the Jodit editor seems to work fine.

I might consider forking the repo with this solution, if anyone is really interested.

**Comment by alainmoran** [comment]
I think there's also a conflict with react-dom too ... I didn't however hit #74 with the versions I was using (NB: you have to force the version for both jodit-react and jodit itself)

**Comment by DaveVanRijn** [comment]
I have submitted a pull request in the Jodit main repo, since the errors originate from that code. Declaring the variables on the global scope fixes the issues, including the HTMLElement conflict.

**Comment by jthorn** [comment]
Any word on when this is going to be released? My app has been broken for weeks because of #74 after after upgrading to resolve it, I was unable to build, so I rolled back. Iv'e been spending the last several days looking for a replacement component, but I'd prefer not to do that.

Also ... is it absolutely necessary to release a new version of jodit-react when an update to jodit is released? It seems like the last few release have simply referenced newer version of jodit without any related code changes. It makes it a bit difficult to review the commit history to determine when something actually changed. I would love to pick on older version of jodit & jodit-react that works, but I have been unable to determine what those would be.

**Comment by DaveVanRijn** [comment]
My pull request has been merged, but as far as I can see it hasn't been published in a release yet. In the meantime I use my own fork which fixes the problem.

You could use the following in your package.json to get your project back up and running until jodit is finally updated:
`"resolutions": {
    "jodit": "git://github.com/DaveVanRijn/jodit.git"
  }`

I do not plan to keep the fork up-to-date with the main repo, so if you use my fork, be sure to keep an eye out for the jodit release.
